### ISSUE TITLE
Server to server token

### ISSUE THREAD

**Issue by matll42** [intuit/QuickBooks-V3-PHP-SDK#503]
Hi, I have a project to interconnect my CRM with quickbook, but I don't want to do any human action to do this sync.

From what I saw, everytime I want to make an action, I have to go through the auhentication process (stored in session).
But in my case I want all action to run server side without human to log in.

Is there a way to achieve it with API Key or something else?

**Comment by SolveASAP** [comment]
You can try accessing through QBWC plugin:
https://developer.intuit.com/app/developer/qbdesktop/docs/get-started/get-started-with-quickbooks-web-connector
I personally created a robot using Python that posts transactions from a spreadsheet at lightning speed
Here is a sample video:
https://www.linkedin.com/posts/solveasap_accounting-automation-robotics-activity-7115803143845642240-CNFi/

**Comment by Warface** [comment]
Isn't the token supposed to be good for 100 days? 

I'm building a plugin on WP to create a form that will create a customer in QB and having to make the handshake between the website and QB for authentification is a pain.

What's the point to have and API with ids and secrets keys that need a modal to make an authentifation on QB each time you need a new token

**Comment by matll42** [comment]
yes, we lost all the interest of the API in case of a server side usage (like me). I had to recreate a manual trigger by user to authenticate him/her so the server can do its process. For that intuit is not that _intuit_ive 😢 
My project is still halfway done, on hold because of that painful authentication
